contracts/anchors/bridged/LinkableAnchorPoseidon2.sol

```diff
@@ -40,7 +40,7 @@ abstract contract LinkableAnchorPoseidon2 is AnchorPoseidon2, ILinkableAnchor {
     bytes32[1] memory history = getLatestNeighborRoots();
     rootHistory[latestHistoryIndex] = history;
     // set the next history index modulo pruning length
-    latestHistoryIndex = latestHistoryIndex % pruningLength;
```

Comment:
afaik this function isn't being used anymore, right? We can remove it now that we have proper tracking of neighbor history and a new collection for the indices.